app/src/shared/components/Toolbar/index.jsx

```diff
@@ -3,33 +3,37 @@
 import React, { type Node } from 'react'
 import cx from 'classnames'
 
-import { Container } from 'reactstrap'
 import Buttons, { type ButtonActions } from '$shared/components/Buttons'
 
 import styles from './toolbar.pcss'
 
 type Props = {
-    status?: Node,
+    left?: Node,
+    middle?: Node,
     actions?: ButtonActions,
     altMobileLayout?: boolean,
 }
 
-const Toolbar = ({ status, actions, altMobileLayout }: Props) => (
+const Toolbar = ({ left, middle, actions, altMobileLayout }: Props) => (
```

Comment:
I'm not a good namer, but maybe this could be more understandable:

left -> leftTools
middle ->middleTools
actions -> rightTools

Reply:
Yep, might be better. I'll keep it for now, the `actions` are fed straight to the `Buttons` component inside the toolbar but that could be left out and go with the above naming.